app/utils/press-release-snapshot.js

```diff
@@ -53,6 +56,7 @@ export default class PressReleaseSnapshot {
     this.pressRelease.governmentFields = this.governmentFields;
     this.pressRelease.themes = this.themes;
     this.pressRelease.sources = this.sources;
+    this.pressRelease.attachments = this.attachments;
```

Comment:
Kan je bij rollback de files die niet opgeslagen worden als attachments (dus de files die in `this.attachments` zitten, maar niet in `this.pressRelease.attachments` vóór de rollback) verwijderen?

Reply:
Aangezien je in de `files-table`-component `deleteRecord` hebt gedaan kan `this.attachments` nu files bevatten in de deleted-state. Deze state moet ook nog ongedaan gemaakt bij een rollback. Je kan dit ongedaan maken door `rollbackAttributes()` aan te roepen op de record (zie ook https://api.emberjs.com/ember-data/3.27/classes/Model/methods/deleteRecord?anchor=deleteRecord)